client/state/selectors/get-jetpack-scan-state.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import 'state/data-layer/wpcom/sites/scan';
+
+/**
+ * Returns the current Jetpack Scan state for a given site.
+ * Returns undefined if the site is unknown, or if no information is available.
+ *
+ * @param {object} state	Global state tree
+ * @param {number} siteId	The ID of the site we're querying
+ * @returns {?object}		An object describing the current scan state
+ */
+export default function getJetpackScanState( state, siteId ) {
+	return get( state.jetpackScan.scan, [ siteId ] );
```

Comment:
Instead of using the get( function we could use the new optional chaining here. 
See p4TIVU-9k9-p2